packages/http/src/mocker/errors.ts

```diff
@@ -0,0 +1,8 @@
+import { Omit } from '@stoplight/types';
+import { ProblemJson } from '../types';
+
+export const INVALID_REQUEST_PAYLOAD: Omit<ProblemJson, 'detail'> = {
+  name: 'INVALID_REQUEST_PAYLOAD',
+  title: 'Invalid request body payload',
+  status: 412,
```

Comment:
Is 412 the right code here? We are talking about validation right? 

422 I would understand, that is common for validation or 400 as we've done before, but 412 is a whole other thing.

Reply:
I'm sorry that was the indeed the intention